src/psyclone/psyir/transformations/omp_target_trans.py

```diff
@@ -88,6 +93,53 @@ class OMPTargetTrans(RegionTrans):
     '''
     excluded_node_types = (CodeBlock, )
 
+    def _add_asynchronicity(self, nodes: List[Node], instance: Directive):
```

Comment:
Is this a duplicate of that in omp_loop_trans.py? Actually, I see that it's not quite the same but could we refactor perhaps?

And/or please update the docstring :-)

Reply:
I think its different enough I preferred for it to have its own implementation.
Added the docstring.